src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -185,12 +190,16 @@ public void setDeadlineTaskFilter(Predicate<? super DeadlineTask> predicate) {
         filteredDeadlineTasks.setPredicate(predicate);
     }
 
+    @Override
+    public void setIsFinishedDeadlineFilter() {
+        setDeadlineTaskFilter(new DeadlineTaskFinishedPredicate());
+    }
```

Comment:
Same goes for all the other methods you added to the `Model` interface just to set a filter.